docs/gcp-instrumentation.md

```diff
@@ -1,8 +1,15 @@
 # Integrating Online Boutique with Google Cloud 
```

Comment:
This is actually not valid anymore.
When adding OTel to the project I haven't updated all the docs, just the main README.
The GCP instrumentation was totally removed from the project, the environment variables were cleaned up and the code was removed from each service.

This file can be safely removed.